code-samples/creating-your-bot/configuration-files/index.js

```diff
@@ -7,7 +7,11 @@ client.once('ready', () => {
 });
 
 client.on('message', message => {
-	if (message.content === '!ping') {
+	if (!message.content.startsWith(config.prefix) {
+		return;
```

Comment:
Personally, i wouldn't use whole block for simple `return;`, neither this format is used further in the guide.
For example: 
https://github.com/discordjs/guide/blob/adef8eae8421f149a4cff148f72d39f3abf3170b/code-samples/creating-your-bot/commands-with-user-input/11/index.js#L9-L10

Reply:
Many apologies, I actually came to the guide after helping someone out with this. I am happy to move to the single-line return, and will do so.